docs/api-docs/getting-started/about-our-api.mdx

```diff
@@ -109,22 +121,31 @@ A media type is the format of the request or response body. BigCommerce APIs acc
 
 ### Requests
 
-#### Request headers
+For more information about required request headers, consult the [API reference](/docs/api) for the endpoint or graph you want to use.
 
-Store Management and Payments API requests require the `Accept`, `X-Auth-Token`, and `Content-Type` headers.
+#### Standard request headers
 
-| Header | Allowed Values | Description | Example |
+| Header | Expected value or type | Description | Example |
 |:-------|:---------------|:------------|:--------|
-| `Accept` | `application/json` (for .json requests) `application/xml` (for .xml requests) | The MIME type format for receiving a response.|`application/xml` |
-| `Content-Type` | `application/json` (for JSON requests) `application/xml` (for XML requests) | The MIME type of the request body. Used to validate and parse the request to the API. | `application/json` |
-| `User-Agent` | String | While it is not required, we ask that you specify a user agent which identifies your integration/client with your requests. |
-| `X-Auth-Token` | String | Access token authorizing the app to access resources on behalf of a user. |
+| `Accept` | MIME types | The MIME type format that indicates which response type the request expects. For more information, see [HTTP Docs: Accept Header (MDN)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept). | `application/json` |
```

Comment:
```suggestion
| `Accept` | MIME types | The MIME type format indicates which response type the request expects. For more information, see [HTTP Docs: Accept Header (MDN)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept). | `application/json` |
```

Reply:
declined, that is used here to do the work of`which`